packages/node/src/index.ts

```diff
@@ -414,6 +415,21 @@ export const build: BuildV3 = async ({
     isEdgeFunction = isEdgeRuntime(staticConfig.runtime);
   }
 
+  const cron = staticConfig?.cron;
+
+  if (cron) {
+    const cronValidationResult = cronValidate(cron, {
+      preset: 'aws-cloud-watch',
```

Comment:
We'll have to remove this one since we're transforming it before putting it on AWS, and to be closer to crontab

Reply:
Yup, updated and moved the validation to the cli, so it applies to all build results, not just node